presto-main/src/main/java/com/facebook/presto/operator/aggregation/builder/InMemoryHashAggregationBuilder.java

```diff
@@ -118,6 +118,12 @@ public InMemoryHashAggregationBuilder(
     @Override
     public void close()
```

Comment:
I added a test using the data provider that's used to test spill, and those tests pass. But I don't know the code enough to tell whether that's because memory was released or it didn't test the scenario we want to test.